Adyen/UI/Form/Items/Text/FormTextInputItemView.swift

```diff
@@ -8,4 +8,20 @@ import Foundation
 
 /// A form view representing a plain text input.
 /// :nodoc:
-public final class FormTextInputItemView: FormTextItemView<FormTextInputItem> {}
+public final class FormTextInputItemView: FormTextItemView<FormTextInputItem> {
+
+    // MARK: - Initializers
+
+    /// Initializes the text item view.
+    /// - Parameter item: The item represented by the view.
+    public required init(item: FormTextInputItem) {
+        super.init(item: item)
+
+        observe(item.$isEnabled) { [weak self] isEnabled in
+            guard let self = self else { return }
+            self.textField.isEnabled = isEnabled
+            isEnabled ? self.updateValidationStatus() : self.resetValidationStatus()
```

Comment:
if we have more than one action for each case of the flag, a regular if else block is more suited IMO.

Reply:
I agree! Changed